### PR TITLE
Feature/save txs flag

### PR DESCRIPTION
**Pull request by cshcomcom**
## Description
- Add saveTxs flag

There are not many cases that tx list is required during testing, so I changed it to be optional.

### CODE REVIEW

File: worker_node/job/confirm.js [hunk 1]
```diff
@@ -27,18 +28,15 @@ class Confirm extends Base {
     const transactionList = [];
     let timeoutTxCount = 0;
     let totalConfirmedTime = 0;
-    const confirmedTimeTable = {}; // TODO(csh): Delete after test
 
     for (let number = from; number <= to; number++) {
       const block = await this.#ain.getBlock(number, true);
+      this.output.statistics.transactionCount += block.transactions.length;
+      if (!this.config.saveTxs) {
```
Comment (liayoo) [inline worker_node/job/confirm.js:35]: Could you add the saveTxs to sample config?

**Review by liayoo — APPROVED** [review]
LGTM with a minor comment. Thanks!

File: worker_node/job/confirm.js (hunk 1 above)
Comment (cshcomcom) [reply worker_node/job/confirm.js:35]: Done! Thanks

**Comment by cshcomcom** [comment]
Thanks for the review! Now merging

**Pull request merged**